pages/index.tsx

```diff
@@ -142,48 +154,72 @@ const Checkout: React.FC<{
       <Grid gutter="xl">
         <Grid.Col md={12} lg={4}>
           <OrderSummary product={customerForm.product} />
+          <Container bg="gray.1" py="xs" mt="sm">
+            <Title mt="lg" mb="sm" order={4}>
+              Widget Options
```

Comment:
Since full screen redirect is not really a "widget". Maybe could call it "Checkout options" or something else